src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        int rand = random.nextInt(5);
+
+        switch (rand) {
+            case 0:
+                return new Circle(10, colorSupplier.getRandomColor());
```

Comment:
A lot of magic numbers here, if figures is random maybe makes sense to make their fields random as well